torch/_functorch/_aot_autograd/traced_function_transforms.py

```diff
@@ -601,7 +615,9 @@ def functional_call(*args, **kwargs):
                         "ignore", "Anomaly Detection has been enabled."
                     )
                     with torch.autograd.detect_anomaly(check_nan=False):
-                        out = Interpreter(mod).run(*args[params_len:], **kwargs)
```

Comment:
Also just a thought - we do this propagation in a few places (e.g. inductor re-runs fake prop), so I guess there's a footgun here than anyone might forget this when writing an retracing pass somewhere, and use the default `fx.Interpreter`.

Do you think that foot gun warrants adding this logic to the default `fx.Interpreter`? (I guess we only really do this once in AOTAutograd and once inductor, so maybe this will come up rarely)

Reply:
I thought about this, and my problem is that... it's kind of PT2 specific so Interpreter feels like the wrong place to put it.